WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainSuggestionsTableViewController.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+class RegisterDomainSuggestionsTableViewController: DomainSuggestionsTableViewController {
```

Comment:
I think this is still on your TODO list, but just so we don't forget - we need to pre-populate domain suggestions based on this comment https://github.com/wordpress-mobile/WordPress-iOS/issues/9630#issuecomment-405507291.

Reply:
yes you are right it is in my TODO list